challengeMk2/ViewModels/StarSystemsViewModel.cs

```diff
@@ -154,8 +82,26 @@ async Task DisplayApiDataAsync()
             }
             finally
             {
+                Title = GetCurrentTitle();

```

Comment:
why calling this also in finally ?

Reply:
When connecting to web Api (catch), while is retrieving data (Sometimes, api is very slow ), the title is
"Retrieving data from web API"
and when data are downloaded (Finally?), the title change to  
"Systems around SOL (API)"